pom.xml

```diff
@@ -596,6 +596,7 @@
     <jms-api-version>2.0.1</jms-api-version>
     <kjetland-mbknor-jsonschema-version>1.0.39</kjetland-mbknor-jsonschema-version>
     <networknt-jsonschmema-validator-version>1.0.66</networknt-jsonschmema-validator-version>
+    <commons-cli-version>1.5.0</commons-cli-version>
```

Comment:
You will see that all of the dependencies are defined in the dependencyManagement section of this file and then can be used without the version in the module-specific pom.

Reply:
Changed.